akka-docs/src/main/paradox/stream/stream-dynamic.md

```diff
@@ -13,21 +13,21 @@ To use Akka Streams, add the module to your project:
 ## Introduction
 
 <a id="kill-switch"></a>
-## Controlling graph completion with KillSwitch
+## Controlling operator completion with KillSwitch
```

Comment:
here I think "stream" would be better, since it's the entire stream that is completed not an individual operator